HW7/NyzkoshapkaOleksandr/task_03.py

```diff
@@ -0,0 +1,5 @@
+def Create_dict_from_string(word = "hello"):
```

Comment:
good idea
look at this implementation
def count_symbols_in_row(word):
    
    result = {}
    
    for item in word:
        if item in result:
            continue
        else:
            result.update({str(item): word.count(item)})
    return result